### DISCUSSION TITLE
Welcome to OMDB-Back-End Discussions!

### DISCUSSION THREAD

**Discussion by cerberusmaster** [cerberusmaster/OMDB-Back-End#1] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cerberusmaster** [comment]
Hello Guys

I have some issues in Laravel development.
How can I set Database Migration?
Could you help me?

**Reply by FFS28** [reply]
Hello @cerberusmaster 

you can use `php artisan make:migration migration_name`.
Please follow this guide.
https://laravel.com/docs/9.x/migrations

**Reply by cerberusmaster** [reply]
Wow!
thanks!

But I have some issues. Because I already made database with several tables.
how can I do implement them?

**Reply by FFS28** [reply]
Hmm...

I think you can make migration per tables.
so you'll define every field of table.
I think

**Reply by cerberusmaster** [reply]
Thanks! Great!

**Comment by cerberusmaster** [comment]
Hello @FFS28 

Could you help me one more?

**Reply by FFS28** [reply]
yes, no problems for me.

**Reply by cerberusmaster** [reply]
Did you have experience in PayPal integration in Laravel?

**Reply by FFS28** [reply]
sure. I have you can use PayPal API document.
